tests/testcore/onebox.go

```diff
@@ -656,6 +659,20 @@ func (c *TemporalImpl) frontendConfigProvider() *config.Config {
 	}
 }
 
+func (c *TemporalImpl) configProvider(serviceName primitives.ServiceName) func() *config.Config {
+	return func() *config.Config {
+		return &config.Config{
+			Services: map[string]config.Service{
+				string(serviceName): {
+					RPC: config.RPC{
+						HTTPPort: int(mustPortFromAddress(c.FrontendHTTPAddress())),
```

Comment:
this is kind of misleading, only frontend has an http port. is this needed for some kind of validation or can it be removed?